docs/recipes/multi-debit-credit-transfers.md

```diff
@@ -62,4 +62,8 @@ In this example, we'll use the following accounts:
 
 Here, we use two transfers to debit accounts `A` and `B` and credit the `Control` account, and another three transfers to credit accounts `X`, `Y`, and `Z`.
 
-If you look closely at this example, you may notice that we could technically have simply debited `B` and credited `Z` directly because the amounts happen to line up. That is true! However, since you probably don't want to program a lot of heuristics to figure out which debits and credits exactly match up, you'll probably find it easier to set up these complex journal entries using this type of control account.
+If you look closely at this example, you may notice that we could technically have simply debited `B` and credited `Z` directly because the amounts happen to line up. That is true!
```

Comment:
```suggestion
If you looked closely at this example, you may have noticed that we could have debited `B` and credited `Z` directly because the amounts happened to line up. That is true!
```

Putting it in the past tense ("If you looked closely") can be a way to surprise people, to get them to pay more attention next time, as if it's a game (like we might quiz them afterwards) compared to asking them to pay more attention now.

Reply:
Removed "technically" and "simply" to make the language itself simpler. Less adverbs, adjectives. Just the raw verbs and nouns. We could still drop "directly" except that illustrates the point.